### PR TITLE
Remove most blocks disabled by `OLD_WAY` if defineds.

### PR DESCRIPTION
**Pull request by schwehr**

### CODE REVIEW

**Review by byrnHDF — APPROVED** [review]

File: hdf/test/litend.c [hunk 1]
```diff
@@ -309,50 +309,6 @@ test_little_read(void)
...
-            for(j=0; j<sizeof(float64); j++)
-              printf("%.2x ",u8_d2[j]);
-            printf("\n");
-}
-              HEprint(stdout,0);
-              num_errs++;
-            } /* end if */
-	  } /* end for */
-}
-#endif /* OLD_WAY */
```
Comment (bmribler) [inline hdf/test/litend.c:355]: Is there a reason to remove both #if and #else blocks?

Comment (derobins) [reply hdf/test/litend.c:355]: This needs to be fixed before merging. We're probably going to yank all the "old machine" stuff (the convex float bits, so this is probably dead code), but let's do that comprehensively in a separate PR.

**Review by derobins — CHANGES_REQUESTED** [review]
Need to fix the one, over-enthusiastic edit

File: hdf/test/litend.c (hunk 1 above)
Comment (schwehr) [reply hdf/test/litend.c:355]: Reverted the whole file and then pushed a change that is hopefully just removing the old code.  Thank you for catching the mistake!

**Review by derobins — APPROVED** [review]

**Comment by derobins** [comment]
Perfect. Let's get rid of this cruft.

**Pull request merged**